product_search/main.py

```diff
@@ -0,0 +1,24 @@
+"""Return the inventory details for an item in the product inventory"""
+
+import sys
+from product_search import (
+    get_product_name,
+    item_inventory_details,
+    display_product_information,
+)
+
+
+def main(filename):
+    """Calls to main functions in product_serach file to retrieve inventory details"""
+    item_name = get_product_name()
+    inv_details = item_inventory_details(item_name)
+    display_product_information(inv_details)
+
+
+if __name__ == "__main__":
+    try:
```

Comment:
This is an excellent pattern. Grab the command line argument and pass it off to `main`. Nice work.